vm/devices/storage/nvme/src/namespace/reservations.rs

```diff
@@ -76,7 +76,7 @@ impl Namespace {
 
         let report_header = nvm::ReservationReportExtended {
             report: nvm::ReservationReport {
-                gen: report.generation,
+                generation: report.generation,
```

Comment:
generators coming toon tm ???

![ItsHappeningGIF](https://github.com/user-attachments/assets/c864295c-ffcc-4b2c-9b29-8f2c7a30946f)

Reply:
Not sure on the status of generators, but at the very least `gen` is being reserved for them. I share your hype.